README.md

```diff
@@ -257,6 +257,7 @@ The contents of index.yaml will be printed to stdout and the program will exit.
 #### Other CLI options
 - `--log-json` - output structured logs as json
 - `--disable-api` - disable all routes prefixed with /api
+- `--disable-statefiles` - disable use of index-cache.yaml to improve caching
```

Comment:
not sure about the usage help: it *improves* caching to remove the statefile ?

Reply:
@davidovich thanks! addressed